### PR TITLE
Adds a special suicide to the Experi-Scanner

### PR DESCRIPTION
**Pull request by Sealed101**
## About The Pull Request
Adds a `suicide_act` to the Experi-Scanner.

<details>
<summary>LiveLeak footage</summary>

https://user-images.githubusercontent.com/75863639/200129319-d112c8ea-e9f6-49a2-872b-8c2625f97e67.mp4

</details>

The victim's brain also gets placed into the resulting toilet's cistern.

<details>
<summary>Bodycam footage from the clean-up crew</summary>

https://user-images.githubusercontent.com/75863639/200129302-4b9b37fe-9c02-4712-8502-e4732b9258f5.mp4

</details>

The resulting toilet will not drop meat sheets when deconstructed, leaving only a pile of bones behind.

_might add some bone crunching sounds later_

## Why It's Good For The Game
More variety is nice. A use for the experiscanner-closed icon. An ultimate sacrifice for a single toilet that you can scan towards an experiment.

## Changelog
:cl:
add: Added a special suicide to the Experi-Scanner
/:cl:

### CODE REVIEW

**Comment by Kubisopplay** [comment]
Final form of every sci main.

**Comment by Ryusenshu** [comment]
The Nex Special

toilet scan

File: code/modules/experisci/handheld_scanner.dm [hunk 1]
```diff
@@ -22,3 +22,37 @@
 	AddComponent(/datum/component/experiment_handler, \
 		allowed_experiments = list(/datum/experiment/scanning, /datum/experiment/physical),\
 		disallowed_traits = EXPERIMENT_TRAIT_DESTRUCTIVE)
+
+/obj/item/experi_scanner/suicide_act(mob/user)
```
Comment (dragomagol) [inline code/modules/experisci/handheld_scanner.dm:26]:
```suggestion
/obj/item/experi_scanner/suicide_act(mob/living/user)
```

Comment (Sealed101) [reply code/modules/experisci/handheld_scanner.dm]: https://github.com/tgstation/tgstation/pull/71103/commits/4d8df75ba0e3e33bee19bb306c0789294f0597b4

**Review by dragomagol — APPROVED** [review]

**Comment by Shadowflame909** [comment]
the most love experiments have gotten in years

**Pull request merged**